core/domain/services/graphql/mutators/BulkEntityDelete.php

```diff
@@ -72,7 +72,7 @@ public static function mutateAndGetPayload()
             $deleted = [];
             $failed  = [];
 
-            foreach ($details['entityDbids'] as $key => $entityDbid) {
+            foreach ($details['entity_db_ids'] as $key => $entityDbid) {
```

Comment:
oops... find and replace error followed by lack of oversight... sry about that :(